src/framework/control.ml

```diff
@@ -617,35 +619,35 @@ struct
                NodeH.modify_opt node join by_node;
              );
            by_loc, by_node
-        in
-
-        let ask ?(node = MyCFG.dummy_node) loc =
-          let f (type a) (q : a Queries.t) : a =
-            match Hashtbl.find_option joined_by_loc loc with
-            | None -> Queries.Result.bot q
-            | Some local -> Query.ask_local_node gh node local q
-          in
-          ({ f } : Queries.ask)
-        in
-
-        (* A node is dead when its abstract value is bottom in all contexts;
-           it holds that: bottom in all contexts iff. bottom in the join of all contexts.
-           Therefore, we just answer whether the (stored) join is bottom. *)
-        let must_be_dead node =
-          NodeH.find_option joined_by_node node
-          (* nodes that didn't make it into the result are definitely dead (hence for_all) *)
-          |> GobOption.for_all Spec.D.is_bot
-        in
-
-        let must_be_uncalled fd = not @@ BatSet.Int.mem fd.svar.vid calledFuns in
-
-        let skipped_statements from_node edge to_node =
-          CfgTools.CfgEdgeH.find_default skippedByEdge (from_node, edge, to_node) []
-        in
-
-        Transform.run_transformations file active_transformations
-          { ask ; must_be_dead ; must_be_uncalled ;
-            cfg_forward = Cfg.next ; cfg_backward = Cfg.prev ; skipped_statements };
+         in
+
+         let ask ?(node = MyCFG.dummy_node) loc =
+           let f (type a) (q : a Queries.t) : a =
+             match Hashtbl.find_option joined_by_loc loc with
+             | None -> Queries.Result.bot q
+             | Some local -> Query.ask_local_node gh node local q
+           in
+           ({ f } : Queries.ask)
+         in
+
+         (* A node is dead when its abstract value is bottom in all contexts;
+            it holds that: bottom in all contexts iff. bottom in the join of all contexts.
+            Therefore, we just answer whether the (stored) join is bottom. *)
+         let must_be_dead node =
+           NodeH.find_option joined_by_node node
+           (* nodes that didn't make it into the result are definitely dead (hence for_all) *)
+           |> GobOption.for_all Spec.D.is_bot
+         in
+
+         let must_be_uncalled fd = not @@ BatSet.Int.mem fd.svar.vid calledFuns in
+
+         let skipped_statements from_node edge to_node =
+           CfgTools.CfgEdgeH.find_default skippedByEdge (from_node, edge, to_node) []
+         in
+
+         Transform.run_transformations file active_transformations
+           { ask ; must_be_dead ; must_be_uncalled ;
+             cfg_forward = Cfg.next ; cfg_backward = Cfg.prev ; skipped_statements };
       );
 
       lh, gh
```

Comment:
All these places where only the indentation is changed should not be place of this PR. it makes the diff hard to review.

Reply:
@michael-schwarz, please see my reply to your other comment regarding this issue.